doc/nrf/ug_zigbee_qsg.rst

```diff
@@ -0,0 +1,216 @@
+.. _ug_zigee_qsg:
+
+Zigbee quick start guide
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This guide is a quick demonstration of some of the basic concepts of the Zigbee network using the |NCS| and the |VSC| extension.
```

Comment:
What about adding link to nRF Connect for Visual Studio Code extension?

Reply:
This is left out on purpose in the first paragraph. I don't want the reader to be distracted with a link to external page.
The "Set up the software" covers installation of VS Code extension.